### ISSUE TITLE
Restore Standby from Repository usng Archive Recovery

### ISSUE THREAD

**Issue by Argha-2890** [pgbackrest/pgbackrest#1001]
Please provide the following information when submitting an issue (feature requests or general comments can skip this):

1. pgBackRest version: 2.24

2. PostgreSQL version: 11

3. Operating system/version - if you have more than one server (for example, a database server, a repository host server, one or more standbys), please specify each: Ubuntu 18.04

4. Did you install pgBackRest from source or from a package? Source

5 Describe the issue: Trying to restore standby DB using Archive recovery instead of Streaming Replication

I want to create a delayed standby node using archive recovery on my existing Patroni cluster. Getting the below error while trying to start the service.

`ERROR: [027]: pg and repo hosts cannot both be configured as remote`

Primary Conf : 

```
[mdb]
pg1-path=/data/patroni1

[global]
log-level-file=detail
repo1-host=10.0.0.24
repo1-host-user=pgbackrest

```

Standby Conf : 

```
[global]
repo1-host=10.0.0.24
repo1-path=/data/pgbackrest
log-level-file=detail

[mdb]
pg1-host=10.0.0.31
pg1-path=/data/patroni3
#recovery-option=standby_mode=on
#recovery-option=primary_conninfo=host=10.0.1.59 user=replicator
#recovery-option=recovery_target_timeline=latest
#recovery-option=recovery_min_apply_delay='1h'

```
Repository Conf : 

```
[global]
repo1-path=/data/pgbackrest
repo1-retention-full=2
start-fast=y

[mdb]
pg1-host=10.0.1.59
pg1-path=/data/patroni1

pg2-host=10.0.0.31
pg2-path=/data/patroni3
```
pgbackrest command : 

pgbackrest --stanza=mdb  --delta restore --log-level-console=info

What I am trying to do is similar to what Gitlab did using Wal-e. https://about.gitlab.com/blog/2019/02/13/delayed-replication-for-disaster-recovery-with-postgresql/

**Comment by dwsteele** [comment]
On the standby you have `pg1-host=10.0.0.31` set which makes pgBackRest think the cluster is not local. Remove that, or specify `--reset-pg1-host` on the command line when you run restore.

**Comment by Argha-2890** [comment]
ok..let me check and get back on this.

**Comment by Argha-2890** [comment]
@dwsteele - Thank you. This is working. Just one more query, I am configuring this node as a delayed replica and just wanted to know whether I need to make any changes in the pgbackrest.conf file to apply recovery_min_apply_delay.

**Comment by dwsteele** [comment]
If you want pgBackRest to automatically write `recovery_min_apply_delay` into `recovery.conf`, then yes. There's an example of using recovery options here: https://pgbackrest.org/user-guide.html#replication/streaming

But this is just to configure PostgreSQL and there's nothing special to do in pgBackRest.  PostgreSQL is in control of the recovery.

**Comment by Argha-2890** [comment]
Apologies for the delay.Thanks for all your support. 👍

**Comment by Argha-2890** [comment]
Hi Team,

I have created a 3 node Patroni cluster, wherein 1 Slave is getting replicated from the Master via pgBackRest archive backup as mentioned in the thread. So, to test failover I stopped the current Master and as expected one of the Slaves got elected as the leader. When I inserted some records in the new Master those changes got reflected on the other Slave as well, despite there being no direct backup process via pgBackRest. So, my question is whether this is happening via Basebackup or some other functionality being implemented.

I know the explanation might be extremely simple however I am unable to gauge the actual cause.
